main_logic/core.py

```
@@ -882,6 +887,70 @@ def _start_tts_thread(self):
+                try:
+                    while not req_queue_ref.empty():
+                        req_queue_ref.get_nowait()
+                except Exception:
+                    pass
+                try:
+                    while not resp_queue_ref.empty():
+                        resp_queue_ref.get_nowait()
+                except Exception:
+                    pass
+
+        # 只在被拆除的 runtime 仍是当前 runtime 时才清全局 TTS 状态，
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)